streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorStateManager.java

```diff
@@ -662,4 +662,10 @@ public TopicPartition registeredChangelogPartitionFor(final String storeName) {
     public String changelogFor(final String storeName) {
         return storeToChangelogTopic.get(storeName);
     }
+
+    public void deleteCheckPointFile() throws IOException {
```

Comment:
nit: I'd suggest we just inline this function inside `StreamTask` since 1) this is only triggered with EOS enabled, and its name `deleteCheckPointFile` maybe a bit misleading, and 2) it is a very simply function anyways.

Reply:
Hi @guozhangwang 

I agree with you that name `deleteCheckPointFile` is a bit misleading, i think we can have a better name ( please suggest a better name :) )

The reasons I had written a new method inside `ProcessorStateManager` is because
-  I felt that, logically; deleting the checkpoint file operation should be under `ProcessorStateManager` as deleting file comes under state management
-  If I have to write this inline, i will have to import `checkpointFile` and `eosEnabled` into `StreamTask` class; but these were already imported in `ProcessorStateManager`, so I created a new method in `ProcessorStateManager`

Please suggest how to go about it.

Reply:
SGTM. We can keep it as is then.